controlpad/controller/controlpad.js

```diff
@@ -44,4 +46,21 @@ export function send_controlpad_message(msg) {
     ws.send(msg);
 }
 
+export function send_datum(msg) {
+    console.log('sending ' + msg);
+    ws.send(msg);
+}
+
+export const MsgType = {
+    "SUBMIT_NAME": 'SUBMIT_NAME'
+}
+
+/**
+ * Send message via websocket
+ * @param {string} type
+ * @param {Record<string, any>} payload
+ */
+export function send_msg(type, payload) {
```

Comment:
Same as first comment about this file becoming a symlink. Handle this in app.js or another file

Reply:
I'll move this